target/linux/octeon/patches-5.4/130-itus_shield_support.patch

```diff
@@ -0,0 +1,42 @@
+--- a/arch/mips/include/asm/octeon/cvmx-bootinfo.h
++++ b/arch/mips/include/asm/octeon/cvmx-bootinfo.h
+@@ -297,7 +297,7 @@ enum cvmx_board_types_enum {
+ 	CVMX_BOARD_TYPE_UBNT_E100 = 20002,
+ 	CVMX_BOARD_TYPE_UBNT_E200 = 20003,
+ 	CVMX_BOARD_TYPE_UBNT_E220 = 20005,
+-	CVMX_BOARD_TYPE_CUST_DSR1000N = 20006,
```

Comment:
I wonder how many hidden users of D-Link DSR-1000 are using OpenWrt? I guess we'll know after this change will be applied :)

Reply:
That's what I also wondered, but how would this be implemented correctly?

Reply:
There's nothing we can do about it, I think this number is set in U-Boot and without flashing new bootloader, You won't change that. I think also dts would be a form of identifying the device in userspace (our scripts) but kernel would still need this id for applying quirks, like the one for PCI.

Reply:
I know that the RHino board I have enums as this, which is why I change it..  Suggestions?

Reply:
Unfortunately no. This can stay as is.

Reply:
@tmn505 Off-topic here, but there is a more recent octeon PR that seems to use device tree at least partially:
https://github.com/openwrt/openwrt/pull/3261
Unfortunately, I have almost no clue how this is working and what the author is doing with it.

Reply:
The Itus Shield never had a device tree.  The kernel contains dtb/dts info for some of the Cavium Octeon lineup (like the CN3xxx/CN5xxxx, but the CN7020 isn't in there.  I also can't speak to Ubiquity's device tree use, if any.